xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -90,13 +91,13 @@ class CDVDMsgAudioCodecChange : public CDVDMsg
 };
 
 
-CVideoPlayerAudio::CVideoPlayerAudio(CDVDClock* pClock, CDVDMessageQueue& parent)
+CVideoPlayerAudio::CVideoPlayerAudio(IVPClockCallback* pIVPClock, CDVDMessageQueue& parent)
```

Comment:
we need to keep this if the interface only implements the functions need by the codec.